src/backend/base/langflow/components/models/baidu_qianfan_chat.py

```diff
@@ -86,15 +101,20 @@ def build_model(self) -> LanguageModel:  # type: ignore[type-var]
         endpoint = self.endpoint
 
         try:
-            output = QianfanChatEndpoint(
-                model=model,
-                qianfan_ak=SecretStr(qianfan_ak).get_secret_value() if qianfan_ak else None,
-                qianfan_sk=SecretStr(qianfan_sk).get_secret_value() if qianfan_sk else None,
-                top_p=top_p,
-                temperature=temperature,
-                penalty_score=penalty_score,
-                endpoint=endpoint,
-            )
+            kwargs = {
+                "model": model,
+                "qianfan_ak": SecretStr(qianfan_ak).get_secret_value() if qianfan_ak else None,
```

Comment:
This can be just `qianfan_ak or None`

Reply:
yes agree, self.qianfan_ak is already a SecretStrInput